File: src/test/java/com/kevin/PersonTest.java
package com.kevin;

import org.junit.Assert;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Assumptions;
import org.junit.jupiter.api.Test;

public class PersonTest {
    @Test
    public void bmiTest(){
        Person person = new Person(67f,1.7f);
        Assertions.assertEquals(67f/(1.7f*1.7f) , person.bmi());
    }
}

File: src/test/kotlin/com/kevin/PersonTest.kt
package com.kevin.kotlin

import com.kevin.Human
import com.kevin.Person
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class PersonTest{
    @Test
    fun humanBmiTest(){
        val human = Human("tom",56f,1.6f)
        Assertions.assertEquals(56f/(1.6f*1.6f) , human.bmi())
    }

    @Test
    fun personTest(){
        val person = Person(60f , 1.78f)
        Assertions.assertEquals(60f/(1.78f*1.78f) , person.bmi())
    }
}